dns_server/Cargo.toml

```diff
@@ -30,5 +30,8 @@ clap = { version = "4", features = ["derive"] }
 directories = "4.0"
 
 [dev-dependencies]
-rstest = "0.16"
+# Local dependencies
```

Comment:
You really don't need to write these comments... stating the obvious isn't usually a good practice in comments.

Reply:
OK, sure, removed.